### PR TITLE
Realease branch env

### PR DESCRIPTION
**Pull request by shrijan-siris**

### CODE REVIEW

File: src/config/env.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require("dotenv").config();
```
Comment (syuraj) [inline src/config/env.js]: @shrijan-siris probably it is a good idea to have separate .env files for dev, qa and prod (as done in our company website using gatsby)
```
require('dotenv').config({
	path: `.env.${process.env.NODE_ENV}`,
})
```

Comment (syuraj) [reply src/config/env.js]: we should try to remove if/else , switch statements coded for each environment

Comment (shrijan-siris) [reply src/config/env.js]: great

**Comment by syuraj** [comment]
@shrijan-siris can you close this pr if it is not needed?

**Pull request closed without merging**